src/builder/Actions.sol

```diff
@@ -28,10 +26,12 @@ import {List} from "./List.sol";
 library Actions {
     /* ===== Constants ===== */
     string constant ACTION_TYPE_BORROW = "BORROW";
+    string constant ACTION_TYPE_MORPHO_BORROW = "MORPHO_BORROW";
```

Comment:
nit: Let's add a TODO to rename "BORROW/REPAY" to "COMET_BORROW" etc.

Reply:
sound good, will do. 👍